<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MartketSim</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="grpBoxCashRegister">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Kassa</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Avaus raja:</string>
     </property>
    </widget>
    <widget class="QLabel" name="value_kassa_auki_raja">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_kassa_auki_raja">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_kassa_auki">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Lis√§ kassojen aukaseminen</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_kassa_sulku">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>161</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Kassojen sulkeminen</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Palvelutiheys:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_palvelutiheys">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_palvelutiheys">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="sliderPosition">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBoxCustomerDensityMultipliers">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>380</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Asiakkakastiheyden kertoimet (%)</string>
    </property>
    <widget class="QSlider" name="slider_kerroin_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>20</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_kerroin_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_kerroin_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>20</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_kerroin_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_kerroin_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>20</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_kerroin_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_kerroin_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>20</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_kerroin_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_kerroin_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>20</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_kerroin_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_kerroin_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>20</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-50</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_kerroin_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBoxCustomers">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Asiakkaat</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Asiakastiheys:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_atiheys">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QSlider" name="slider_atiheys">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="sliderPosition">
      <number>4</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpBoxCharts">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Kuvaaja</string>
    </property>
    <widget class="QComboBox" name="comboBox_kuvaaja">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Simuloi</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>661</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionEnglish"/>
    <addaction name="actionFinnish"/>
   </widget>
   <addaction name="menuLanguage"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionFinnish">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Finnish</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
